pkg/dns/dns.go

```diff
@@ -33,6 +32,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	kcache "k8s.io/kubernetes/pkg/client/cache"
 	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	"k8s.io/kubernetes/pkg/dns/util"
```

Comment:
go compiler will flag if there is a name conflict, I would prefer to not explicitly name unless it is required.